src/interface/TWAsnParser.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright © 2017-2023 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include <TrustWalletCore/TWAsnParser.h>
+
+#include "AsnParser.h"
+
+using namespace TW;
+
+TWData *_Nullable TWAsnParserEcdsaSignatureFromDer(TWData *_Nonnull encoded) {
```

Comment:
Would it be better to put it in https://github.com/trustwallet/wallet-core/blob/master/src/WebAuthn.cpp? I'm ok with any option

Reply:
I think AsnParser can be used not only in the WebAuthn, such as Base58, Base32 etc. So probably it will be helpful for someone too 🤔